addons/sale/tests/test_sale_tax_totals.py

```diff
@@ -1,15 +1,20 @@
 # -*- coding: utf-8 -*-
-from odoo.addons.account.tests.test_invoice_tax_totals import TestTaxTotals
+from odoo.addons.sale.tests.common import TestSaleCommonBase
+from odoo.fields import Command
 from odoo.tests import tagged
 
 
 @tagged('post_install', '-at_install')
-class SaleTestTaxTotals(TestTaxTotals):
+class SaleTestTaxTotals(TestSaleCommonBase):
```

Comment:
Noticing now, but nope, please use another test class/file to add the new test.
This one is a specific test class that tests `accounting` logic, with the test methods being defined in `account`.
It's horrifyingly architectured, but I think it works, so don't touch it.